testdata/go/v4/memcached-operator/Makefile

```diff
@@ -48,7 +48,7 @@ endif
 
 # Set the Operator SDK version to use. By default, what is installed on the system is used.
 # This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
-OPERATOR_SDK_VERSION ?= v1.35.0
+OPERATOR_SDK_VERSION ?= v1.36.0
```

Comment:
This need not change right now, please revert.

Reply:
Done. I did it because of this failure: https://github.com/operator-framework/operator-sdk/actions/runs/10270485781/job/28418909748

Reply:
I'm honestly not sure when those are supposed to be updated (pre or post release), since this is not part of the release process docs. I've pushed a PR that is merged to address this, so you can rebase off of main now.

Reply:
I've rebased my changes